internal/service/resources/api/openapi.yaml

```diff
@@ -667,6 +667,7 @@ components:
             List of globalLocationIDs that were assigned to the OCloud Site(s) which this Deployment Manager supports.
           items:
             type: string
+            format: uuid
```

Comment:
the spec doesn't say this is a uuid - it explicitly says string, not identifier

Reply:
Yes, I know but it sort of sounds like this is supposed to be a collection of globalLocationId values from the pools which isn't well defined in the spec and sounds like it is a uuid...  the conformance tests seem to treat these as non-uuid strings so i'll line them both up the same way for now.

Reply:
Fixed